soccer_hardware/launch/soccer_hardware.launch

```diff
@@ -1,9 +1,27 @@
 <launch>
+    <arg name="robot_name" default="robot1" />
```

Comment:
add argument called test_walking, and enable complementary filter and robot_pose_ekf if test_walking is true